server/lib/referenceLookup.js

```diff
@@ -59,3 +78,31 @@ function indexIdentifiers() {
   });
 }
 
+// hack: handle special case of stock movements
+function stockMovementType(id) {
+  /**
```

Comment:
I can not understand theses comments, why they are there?

Reply:
Normally this file was for handling just reference of this format : `Prefix.Project.Reference` but stock movement has the format : `Prefix.FluxId.Reference`.